pages/premium.php

```diff
@@ -578,12 +578,36 @@ function premium() {
                 // Confirm change in seal... time will not be reimbursed
                 if(!isset($_POST['change_forbidden_seal'])) {
                     $confirmation_type = 'change_forbidden_seal';
-                    $confirmation_string = "Are you sure you would like to change from your {$player->forbidden_seal->name}?<br />
-                    You will lose {$system->time_remaining($player->forbidden_seal->seal_time_remaining)} of premium time.<br />
-                    <b>This can not be undone!</b>";
+                    if ($player->forbidden_seal->name == "Twin Sparrow Seal") {
+                        // Convert remaining premium time to days and calculate AK value
+                        $akCredit = floor(floor($player->forbidden_seal->seal_time_remaining / 86400) / 6);
+                        // Adjust purchase cost with minimum 0
+                        if (($akCost -= $akCredit) < 0) {$akCost = 0;}
+                        $confirmation_string = "Are you sure you would like to change from your {$player->forbidden_seal->name}?<br />
+                        You will lose {$system->time_remaining($player->forbidden_seal->seal_time_remaining)} of premium time.<br />
+                        Up to {$akCredit} Ancient Kunai will be credited toward your purchase from existing premium time.<br />
+                        <b>This can not be undone!</b>";
+                    }
+                    else if ($player->forbidden_seal->name == "Four Dragon Seal") {
```

Comment:
Similar to above with Twin seal.

Reply:
Added second commit simplifying per Hitori's suggestions.